storage/innobase/row/row0sel.cc

```diff
@@ -2987,8 +2987,6 @@ bool row_sel_store_mysql_rec(byte *mysql_rec, row_prebuilt_t *prebuilt,
     mem_heap_empty(blob_heap);
   }
 
-  if (UNIV_LIKELY_NULL(prebuilt->compress_heap))
-    row_mysql_prebuilt_free_compress_heap(prebuilt);
```

Comment:
Can you plz explain the following?
1. How it is reused? This function is supposed to transfer/convert a innodb record to server/mysql record
2. When is it freed?
3. How about emptying it? Like blob_heap above. Can't we do the same?

Reply:
Can you please bring this back and test? Only then we will know if your new fix is working or not?